test/inductor/test_pattern_matcher.py

```diff
@@ -664,6 +665,37 @@ def fn2(inp, a, b):
         _, (code) = run_and_get_code(fn2, args[0], args[1], args[2])
         FileCheck().check_not("extern_kernels.addmm(").run(code[0])
 
+    def test_match_equivalent_function_invocations(self):
+        # pad_mm.py only registered `addmm_pattern` which explicitly passes in alpha and beta,
+        # since they have default values, we should also match the equivalent invocations like
+        # the following two cases.
+        @torch.compile()
```

Comment:
Can we also add a test in the other direction, say where the user registers `addmm` without alpha and beta, but alpha=1. and beta=1. are passed in? 

And where the user registers a pattern with just alpha handled but not beta ?

Note: torch_dispatch will remove default args, so we might need to register passes that would add/remove them from node to exercise our logic